pkg/dns/azure/dns_test.go

```diff
@@ -96,3 +99,55 @@ func TestDeleteDNS(t *testing.T) {
 		t.Fatalf("expected the dns client 'Delete' func to be called, but found %s instead", recordedCall)
 	}
 }
+
+func TestGetTagList(t *testing.T) {
```

Comment:
All new tests for specific functions should name tests following the pattern "Test_foo" for the function named "foo".
```suggestion
func Test_GetTagList(t *testing.T) {
```

Reply:
incorporated comment along with the other comments.